src/lib/utils/FileFactory.ts

```diff
@@ -20,8 +20,6 @@ export class FileFactory {
             let sourcePath = path.resolve(path.join(this.sourcePath, fileName));
             let fileContents = fs.readFileSync(sourcePath, 'utf8');
             let destPath = this.frameworkFiles[fileName];
```

Comment:
@georgejecook There's no need to use the AST editor because this is a new file and isn't editing user code.